.github/workflows/promote_charm_haproxy_spoe_auth.yaml

```
@@ -0,0 +1,31 @@
+name: Promote the haproxy-spoe-auth-operator charm
+
+on:
+  workflow_dispatch:
+    inputs:
+      origin-channel:
+        type: choice
+        description: 'Origin Channel'
+        options:
+        - latest/edge
+      destination-channel:
+        type: choice
+        description: 'Destination Channel'
+        options:
+        - latest/stable
+    secrets:
+      CHARMHUB_TOKEN:
+        required: true
+
+jobs:
+  promote-charm:
+    uses: canonical/operator-workflows/.github/workflows/promote_charm.yaml@main
+    with:
+      origin-channel: ${{ github.event.inputs.origin-channel }}
+      destination-channel: ${{ github.event.inputs.destination-channel }}
+      working-directory: ./haproxy-spoe-auth-operator
+      base-channel: 24.04
+      base-name: ubuntu
+      base-architecture: amd64
+      tag-prefix: haproxy-spoe-auth-operator
```

Maybe we can be less verbose? No strong opinion, just and idea.
```suggestion
      tag-prefix: haproxy-spoe-auth
```